src/main/java/org/iban4j/bban/BbanStructure.java

```diff
@@ -34,6 +34,13 @@ private BbanStructure(final BbanStructureEntry... entries) {
 
     private static final EnumMap<CountryCode, BbanStructure> structures;
 
+    /* French sub-territories may use their own country code (BL,RE,NC,...) or FR for their IBAN. Structure is the same, only the IBAN checksum differ. */
+    private final static BbanStructure FRENCH_STRUCTURE = new BbanStructure(
+        BbanStructureEntry.bankCode(5, 'n'),
+        BbanStructureEntry.branchCode(5, 'n'),
+        BbanStructureEntry.accountNumber(11, 'c'),
+        BbanStructureEntry.nationalCheckDigit(2, 'n'));
+
```

Comment:
Why did you add new structure here? As for as I see this is the same structure as CountryCode.FR.
You could make use of it and did not break the 2 letter abbreviation standard.
Is there any reason behind this?

Reply:
What kind of solution would you prefer/expect?